src/fable-library/Types.ts

```diff
@@ -35,15 +35,19 @@ export function declare(cons: any, superClass?: any) {
 export class SystemObject implements IEquatable<any> {
 
   public toString() {
-    return "{" + Object.entries(this).map(([k, v]) => k + " = " + String(v)).join(";\n ") + "}";
+    return this.ToString();
+  }
+
+  public ToString() {
+    return this.constructor.name;
   }
 
-  public GetHashCode(x?: any) {
-    return identityHash(x ?? this);
+  public GetHashCode() {
```

Comment:
Ah, ok. So that was the reason! I didn't know why it was like that and I wanted to clean the interfaces so I removed, but it should be ok to put it back :+1: